Google.PowerShell/Sql/GcSqlCmdlet.cs

```diff
@@ -68,7 +68,9 @@ public Operation WaitForSqlOperation(Operation op)
             Random rnd = new Random();
             int randProc = rnd.Next();
             ProgressRecord progress =
-                new ProgressRecord(randProc, $"Waiting for operation '{op.OperationType}' to finish.", "Waiting");
+                new ProgressRecord(randProc,
```

Comment:
Line break after `(` so `randProc` is on the next line, like the other parameters.